danielMunteanu/src/main/java/Order.java

```diff
@@ -0,0 +1,67 @@
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.List;
+
+class Order {
+    public long id;
+    int counter;
+    public String date;
```

Comment:
if we are using the getters and setters then we can limit access to this properties by using PRIVATE access modifier

Reply:
agree with that, but I left them public as said in the task

Reply:
Pay attention to the type of date variable

Reply:
fixed in next commit